x/programs/cmd/simulator/src/lib.rs

```diff
@@ -85,9 +85,8 @@ pub enum Key {
 
 #[derive(Clone, Debug, PartialEq, Default, Serialize)]
 #[serde(rename_all = "camelCase")]
-#[non_exhaustive]
 pub struct TestContext {
-    program_id: Id,
+    pub program_id: Id,
```

Comment:
Without these changes, it was not possible to "autofill" a `TestContext` from another one. Pretty useful in the the case of the `voter_context`, see the multisig test `cast_votes` for more information.